frontend/src/app/utils/srGraphQLErrorMessage.tsx

```diff
@@ -23,32 +23,54 @@ export function getHeader(message: string): string {
   return getErrorText(message, "header");
 }
 
+function createSupportMailToLink(msg: string, index: number) {
+  return (
+    <span key={index}>
+      {" "}
+      <a href="mailto:support@simplereport.gov">{msg}</a>{" "}
+    </span>
+  );
+}
+
+function createContactUsLink(msg: string, nextMsg: string, index: number) {
+  return (
+    <span key={index}>
+      {" "}
+      <a
+        target="_blank"
+        rel="noopener noreferrer"
```

Comment:
Thank you for flagging that! Resolves: https://github.com/CDCgov/prime-simplereport/pull/5165#discussion_r1094800905